examples/webgl_pmrem_test.html

```diff
@@ -0,0 +1,184 @@
+                <br>Mouse-over is a PMREM of the skybox: a single bright pixel representing the same directional light source.
+                <br>The difference between these renders indicates the error in the PMREM approximations.
+			</div>
+		</div>
+
+		<script type="module">
+
+			import * as THREE from '../build/three.module.js';
+
+            import { OrbitControls } from './jsm/controls/OrbitControls.js';
+            import { RGBELoader } from './jsm/loaders/RGBELoader.js';
+			import { PMREMGenerator } from './jsm/pmrem/PMREMGenerator.js';
+
+            var scene, camera, controls, renderer;
+            
+			function init() {
+
+				var width = window.innerWidth;
+				var height = window.innerHeight;
+				var aspect = width / height;
+
+				// renderer
+
+				renderer = new THREE.WebGLRenderer( { antialias: true } );
+				renderer.setSize( width, height );
+                renderer.setPixelRatio( window.devicePixelRatio );
+                renderer.gammaOutput = true;
+                renderer.physicallyCorrectLights = true;
+				document.body.appendChild( renderer.domElement );
+
+                window.addEventListener( 'resize', onResize, false );
+                
+				// scene
+
+				scene = new THREE.Scene();
+
+				// camera
+
+				camera = new THREE.PerspectiveCamera( 40, aspect, 1, 30 );
+                camera.position.set( 0, 0, 18 );
+
+                // controls
+                
+                controls = new OrbitControls( camera, renderer.domElement );
+                controls.update();
+
+                // light
+                
+                var directionalLight = new THREE.DirectionalLight( 0xffffff, 1 );
+                var x = 597;
+                var y = 213;
+                var theta = ( x + 0.5 ) * Math.PI / 512;
+                var phi = ( y + 0.5 ) * Math.PI / 512;
+
+                directionalLight.position.set(
+                    - Math.sin( phi ) * Math.cos( theta ),
+                    Math.cos( phi ),
+                    - Math.sin( phi ) * Math.sin( theta ) );
+
+                scene.add( directionalLight );
+
+				document.body.addEventListener( 'mouseover', function () {
+
+					scene.traverse( function ( child ) {
+
+						if ( child.isMesh ) {
+
+                            child.material.envMapIntensity = 1;
+                            directionalLight.intensity = 0;
+                        
+                        }
+
+					} );
+
+				} );
+
+				document.body.addEventListener( 'mouseout', function () {
+
+					scene.traverse( function ( child ) {
+
+						if ( child.isMesh ) {
+
+                            child.material.envMapIntensity = 0;
+                            directionalLight.intensity = 1;
+
+                        }
+
+					} );
+
+                } );
+
+			}
+
+			function createObjects() {
+
+                var radianceMap = null;
+                new RGBELoader()
+                    .setDataType( THREE.UnsignedByteType )
```

Comment:
I did this so I could read the values:
```js
.setDataType( THREE.FloatType )
```
But you can add it and comment it out if you want. Or not add it. :-)

Reply:
Added as a comment.